common/ntp/ntp.go

```diff
@@ -0,0 +1,35 @@
+package ntp
+
+import (
+	"time"
+
+	beevik_ntp "github.com/beevik/ntp"
+)
+
+const (
+	// tolerance range of local clock with NTP time server
+	toleranceRange = time.Duration(50 * time.Millisecond)
```

Comment:
How did we come with this tolerance? I am actually not sure whether this 50 milliseconds works in general, but Ethereum has this clock drift threshold with only 10 seconds. (go-ethereum/p2p/discover/v4_udp.go:59)

Reply:
yea, we could tolerant seconds level too, as our view change window is 45 seconds.

Reply:
sure. we can have two levels of tolerance levels.